config/intellij-idea-inspections.properties

```diff
@@ -2,7 +2,10 @@
 # Content should be the same as in TeamCity "Include / exclude patterns:"
 idea.exclude.patterns=.idea/**;\
               .ci/*.config;\
-              .ci/pitest-survival-check-xml.groovy\
+              .ci/checker-framework.groovy;\
+              .ci/codenarc.groovy;\
+              .ci/error-prone-check.groovy;\
+              .ci/pitest-survival-check-xml.groovy;\
```

Comment:
Curious, what happens if we run this on groovy scripts? Is this a future possibility / issue?

Reply:
Yes, we get a few weird violations due to not having groovy configured correctly (I think it is a language level issue)